### PR TITLE
Reviewed unit tests of rdkit.VLib - coverage now 67.1%

### PR DESCRIPTION
**Pull request by gedeck**
Most modules are at 100%, only the Database modules are untested.

### CODE REVIEW

File: rdkit/VLib/Filter.py [hunk 1]
```diff
@@ -3,10 +3,10 @@
 #  Copyright (C) 2003 Rational Discovery LLC
 #     All Rights Reserved
 #
-import sys
+
+import six
```
Comment (bp-kelley) [inline rdkit/VLib/Filter.py]: Why the change from rdkit.six to six?

Comment (gedeck) [inline rdkit/VLib/Filter.py]: I find it a bit strange to have an internal copy of six (and even modify it). However, I don't feel to strongly about this and have reverted these changes.

Comment (rvianello) [inline rdkit/VLib/Filter.py]: the reasons for a bundled six module are mainly related to this small customization

https://github.com/rdkit/rdkit/blob/master/rdkit/six.py#L209

that was helpful to address some portability issues we faced while unpickling some data files used by the test code.

to a very much lesser extent, it also helps avoiding an additional external dependency

Comment (greglandrum) [inline rdkit/VLib/Filter.py]: Yes, this is the important point for me: why add the external dependency when we don't need it.

Comment (gedeck) [inline rdkit/VLib/Filter.py]: All changes reversed. I wonder if there is a more obvious way to document the changes to the original six module. I will think about this and propose an alternative if I find a reasonable one.

File: rdkit/VLib/NodeLib/UnitTestNodeLib.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# $Id$
```
Comment (greglandrum) [inline rdkit/VLib/NodeLib/UnitTestNodeLib.py]: Looks like you added two new test files without updating test_list.py.

Comment (gedeck) [inline rdkit/VLib/NodeLib/UnitTestNodeLib.py]: The two UnitTest files run the doc tests as well. I now removed them from the test_list.py files and added the UnitTestNodeLib and UnitTestVLib files instead.
